src/ansys/templates/python/osl_solution/hooks/post_gen_project.py

```diff
@@ -6,7 +6,7 @@
 
 
 DESIRED_STRUCTURE = [
-    ".github/workflows/ci.yml",
+    ".github/workflows/build-release.yml",
     ".vscode/launch.json",
```

Comment:
@nick-marnik I don't know what is the best naming here, but in the baseline solution we use ci.yml. So we need to come up with a single naming convention.

Reply:
I can't find the previous comment I made, but I have changed this in a few different solutions recently. My argument is that any file in `.github/workflows` is inherently related to CI/CD. My intention is to provide a more descriptive filename that gives some context for what the workflow does.

I akin this to things like `var myVar` or `bool myBool` - it is just repetitive information.

I'm open to suggestions if you think a different filename is better.